tests/coafileTest.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+import re
+from coalib.misc.Shell import run_shell_command
+from coalib.settings.ConfigurationGathering import load_config_file
+
+
+class coafileTest(unittest.TestCase):
+
+    def setUp(self):
+        self.filepath = '.coafile'
+        self.sections = load_config_file(self.filepath, None, True)
+
+    def test_commit_regex(self):
+        shortlog_regex = self.sections['commit']['shortlog_regex'].value
+        git_command = 'git log -100 --pretty=%s'
+        git_log, stderr = run_shell_command(git_command)
+        self.assertEqual(stderr, '', msg=stderr)
```

Comment:
assertFalse.
`msg=stderr` is unhelpful. the value of stderr will be shown anyway.